tests/backup/backup_rbac_test.go

```diff
@@ -1520,7 +1513,11 @@ var _ = Describe("{VerfiyRBACforAppUser}", func() {
 	})
 
 	JustAfterEach(func() {
-		defer EndPxBackupTorpedoTest(scheduledAppContexts)
+		defer func() {
+			err := SetSourceKubeConfig()
+			log.FailOnError(err, "Unable to switch context to source cluster [%s]", SourceClusterName)
+			EndPxBackupTorpedoTest(scheduledAppContexts)
+		}()
```

Comment:
How about?

```
		defer EndPxBackupTorpedoTest(scheduledAppContexts)
		defer func() {
			err := SetSourceKubeConfig()
			log.FailOnError(err, "Unable to switch context to source cluster [%s]", SourceClusterName)
		}()
```

If switching context failed I think EndPxBackupTorpedoTest will not run.

Reply:
Addressed the change